lib/browser/api/web-contents.js

```diff
@@ -325,6 +337,22 @@ WebContents.prototype.getZoomFactor = function (callback) {
   })
 }
 
+const wrapEventWithReply = (event, internal) => {
```

Comment:
split this into two functions, `wrapEventWithReply` and `wrapEventWithReplyInternal`.

... also, this doesn't actually "wrap" the event, so maybe `addReplyToEvent` might make more sense?